image.adoc

```diff
@@ -1,43 +1,62 @@
-## Image Service API
 
-### Service Description
-Image Service API allows uploading of product images to be stored and used within the iZettle applications.
-The service returns a `imageUrl` that can be added to a desired product using the Product Library API.
-The images can be uploaded using a byte array or a static URL, the image will be eventually available for lookup from
-within the iZettle apps or via the `imageUrls` returned by the service (See Swagger documentation for upload restrictions).
+= Image API
 
-`NOTE: JPG does not exist as an imageFormat, you must use JPEG for .jpg images`
+== Description
+====
+Image API allows upload of product images to be stored and used within iZettle applications.
+The API returns an `imageUrl` that can be added to a desired product using the Product Library API.
+Images can be uploaded using a byte array or a static URL, the image will be eventually available for look up from
+within iZettle apps or via `imageUrls` returned by the API.
 
-### URL
-https://image.izettle.com/v2/images
+NOTE: JPG does not exist as `imageFormat`, you must use "JPEG" for .jpg images.
```

Comment:
**Note:**
Question: why is this note displayed in table cells? Sorry I didn't figure out what tagging creates table cells here.

Reply:
I think Github styles it differently.